README.md

```diff
@@ -1,140 +1,146 @@
 MiraOS for RuuviTag
 ===================
+This repo provides a guide for setting up a sensor network using Mira products and RuuviTags. Sensor data is stored in InfluxDB and displayed via a Grafana dashboard.
 
-An example of running the MiraOS on RuuviTag hardware, to show how commissioning
-and status can be made using NFC
+# Overview
+RuuviTags form a Mira mesh network, transmitting sensor data to a border gateway. The gateway is a Raspberry Pi equipped with a MiraUSB, which serves as the network's root and acts as a Radio on a Stick (RoaS). The mira-gateway systemd service manages communication with the MiraUSB and directs traffic into the host's network stack. On the host, three containerized services operate:
 
-Build and installation
-----------------------
+1. InfluxDB: A database for sensor data storage.
+2. udp-to-influx: Processes packets from the Mira gateway and saves the sensor data to InfluxDB.
+3. Grafana: Retrieves and visually represents data in a dashboard.
+
+# Quick start
+
+The following components are needed.
+
+1. libmira
+2. MiraUSB
+3. Raspberry Pi 3 or 4 with a SD card and a power supply
+4. A couple of Ruuvi tags
+5. Mira licenses for the Ruuvi tags
+
+To get any of the Mira products, please get in contact with sales@lumenradio.com.
+## Setup raspbian
+1. Follow the instructions on https://www.raspberrypi.com/software/ to create a new installation of Raspebrry Pi OS.
+2. Connect your raspberry Pi to the internet. 
+3. Create a sudo user and log in. 
+
+## Install docker
+```
+curl -sSL https://get.docker.com | sh
+sudo usermod -aG docker <username>
+```
+
+## Set up influxdb, grafana and udp-to-influx services
+Copy the mira-ruuvitag/RPi directory the the Raspberry Pi.
+```
+scp -r mira-ruuvitag/RPi <username>@<raspberry_pi_ip>:/home/<username>/
```

Comment:
This step should only be done if the setup is done remotely.

Reply:
Good point. 

So add one version with steps on the RPi or just assume the user is working on the RPi?

Reply:
I think one can assume that the user is working on the RPi directly.

Reply:
fixed